.gitignore

```diff
@@ -86,6 +86,7 @@ CMakeFiles
 CMakeScripts
 Testing
 Makefile
+!nitro-node/Makefile
```

Comment:
Why ignore then unignore?

Reply:
Makefile that is generated by Cmake for nitro (cpp) is ignored from root (can alsway specify the output build directory when invoking Cmake). But I need to put some utilities for CI workflow in `nitro-node/Makefile`. For now, just use Makefile for CI tasks definition for `@janhq/nitro-node` until I can find a better replacement.